src/DataWrangling/Copernicus/Copernicus.jl

```diff
@@ -78,12 +94,39 @@ copernicus_dataset_variable_names = Dict(
     :free_surface => "zos",
 )
 
+copernicus_bgc_daily_dataset_variable_names = Dict(
+    :total_chlorophyll => "chl",
+    :primary_production => "nppv",
+    :nitrate => "no3",
+    :phosphate => "po4",
+    :dissolved_silicate => "si",
+    :dissolved_oxygen => "o2",
+    :dissolved_iron => "fe",
+    :ph => "ph",
+    :surface_co2 => "spCO2",
+    :total_phytoplankton => "phyc",
+)
+
+copernicus_bgc_daily_dataset_variable_names = Dict(
+    :total_chlorophyll => "chl",
+    :primary_production => "nppv",
+    :nitrate => "no3",
+    :phosphate => "po4",
+    :dissolved_silicate => "si",
+    :dissolved_oxygen => "o2",
+)
```

Comment:
It looks possible to lump all the names into a single "main dictionary" for all `CopernicusMetadata` --- should we do that? happy to keep them separate if needed or if its better that way.

Reply:
hm.. I think the `available_variables` won't work with that suggestion...

Reply:
I initially lumped them together and then thought it might make more sense to separate so it's not on the user to know which variables are in which data product (and because I saw there are separate dictionaries in ECCO.jl for ECCO2 and ECCO4).

Reply:
> hm.. I think the `available_variables` won't work with that suggestion...

can you clarify?